Sources/NIOIRC/IRCMessageParser.swift

```diff
@@ -135,13 +136,37 @@ public struct IRCMessageParser {
       return String(data: Data(slice), encoding: .utf8) // Sigh, the pain.
     }
 
+    /* parse metadata */
+
+      let metadataSource : Swift.Slice<UnsafeRawBufferPointer>?
+
+      if cursor[cursor.startIndex] == cAt {
+        let startIndex = cursor.startIndex.advanced(by: 1)
+        let spaceIdx   = line.firstIndex(of: cSpace)
```

Comment:
Is it safe to assume that the metadata itself doesn't contain spaces? Is there some escaping maybe?